2018-10-31T14:43:08.866Z - [32minfo[39m: Data Base connection success ...
2018-10-31T14:43:08.944Z - [31merror[39m: uncaughtException: expected { Object (domain, _events, ...) } to have status code 401 but got 200 date=Wed Oct 31 2018 18:43:08 GMT+0400 (+04), pid=13517, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=155099136, heapTotal=120229888, heapUsed=92331840, external=183467, loadavg=[0.57373046875, 0.79150390625, 0.83935546875], uptime=118089, trace=[column=33, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.get.end, line=56, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { Object (domain, _events, ...) } to have status code 401 but got 200,     at chai.request.get.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:56:33),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-10-31T14:43:08.955Z - [31merror[39m: uncaughtException: expected { Object (domain, _events, ...) } to have status code 403 but got 200 date=Wed Oct 31 2018 18:43:08 GMT+0400 (+04), pid=13517, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=155369472, heapTotal=120229888, heapUsed=94925104, external=183933, loadavg=[0.57373046875, 0.79150390625, 0.83935546875], uptime=118089, trace=[column=33, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.get.set.end, line=68, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { Object (domain, _events, ...) } to have status code 403 but got 200,     at chai.request.get.set.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:68:33),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-10-31T14:43:08.988Z - [31merror[39m: uncaughtException: expected { Object (domain, _events, ...) } to have status code 204 but got 200 date=Wed Oct 31 2018 18:43:08 GMT+0400 (+04), pid=13517, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=155639808, heapTotal=121278464, heapUsed=98080576, external=195130, loadavg=[0.57373046875, 0.79150390625, 0.83935546875], uptime=118089, trace=[column=33, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.put.end, line=154, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { Object (domain, _events, ...) } to have status code 204 but got 200,     at chai.request.put.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:154:33),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-10-31T14:43:09.006Z - [31merror[39m: uncaughtException: expected { Object (domain, _events, ...) } to have status code 404 but got 200 date=Wed Oct 31 2018 18:43:09 GMT+0400 (+04), pid=13517, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=160776192, heapTotal=126521344, heapUsed=90279240, external=83520, loadavg=[0.57373046875, 0.79150390625, 0.83935546875], uptime=118089, trace=[column=33, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.put.send.end, line=169, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { Object (domain, _events, ...) } to have status code 404 but got 200,     at chai.request.put.send.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:169:33),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-10-31T14:43:09.015Z - [31merror[39m: uncaughtException: expected { status: false, platform: 'magento' } to have property 'status' of true, but got false date=Wed Oct 31 2018 18:43:09 GMT+0400 (+04), pid=13517, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=160776192, heapTotal=126521344, heapUsed=90847472, external=92720, loadavg=[0.57373046875, 0.79150390625, 0.83935546875], uptime=118089, trace=[column=44, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.del.send.end, line=205, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { status: false, platform: 'magento' } to have property 'status' of true, but got false,     at chai.request.del.send.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:205:44),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-10-31T14:43:09.019Z - [31merror[39m: uncaughtException: expected { status: false } to equal {} date=Wed Oct 31 2018 18:43:09 GMT+0400 (+04), pid=13517, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=160776192, heapTotal=126521344, heapUsed=91195752, external=93202, loadavg=[0.57373046875, 0.79150390625, 0.83935546875], uptime=118089, trace=[column=44, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.del.send.end, line=215, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { status: false } to equal {},     at chai.request.del.send.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:215:44),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
